frontend/src/components/common/Resource/EditorDialog.tsx

```diff
@@ -109,6 +112,15 @@ export default function EditorDialog(props: EditorDialogProps) {
 
   const theme = useCurrentAppTheme();
 
+  const [hideManagedFields, setHideManagedFieldsState] = React.useState(() => {
```

Comment:
we have a `useLocalStorageState` hook that you could use here instead

Reply:
used.